uaclient/cli.py

```diff
@@ -204,8 +204,11 @@ def action_enable(args, cfg):
     @return: 0 on success, 1 otherwise
     """
     print(ua_status.MESSAGE_REFRESH_ENABLE)
-    if not contract.request_updated_contract(cfg):
-        logging.debug(ua_status.MESSAGE_REFRESH_FAILURE)
+    try:
+        contract.request_updated_contract(cfg)
+    except (util.UrlError, exceptions.UserFacingError) as exc:
+        # Inability to refresh is not a critical issue during enable
+        logging.debug(ua_status.MESSAGE_REFRESH_FAILURE + ". " + str(exc))
```

Comment:
As this is debug level (and therefore won't go to the console by default), we can just pass `exc_info=True` to get the full traceback in the log.